internal/testutils/xds/e2e/setup_management_server.go

```diff
@@ -99,14 +101,69 @@ func DefaultBootstrapContents(nodeID, serverURI string) ([]byte, error) {
 	}
 
 	// Create the bootstrap configuration.
-	bs, err := bootstrap.Contents(bootstrap.Options{
+	bs, err := bootstrap.NewContentsForTesting(bootstrap.ConfigOptionsForTesting{
+		Servers: []json.RawMessage{[]byte(fmt.Sprintf(`{
+			"server_uri": %q,
+			"channel_creds": [{"type": "insecure"}]
+		}`, serverURI))},
 		NodeID:                             nodeID,
-		ServerURI:                          serverURI,
 		CertificateProviders:               cpc,
 		ServerListenerResourceNameTemplate: ServerListenerResourceNameTemplate,
+		Authorities: map[string]json.RawMessage{
+			// Most tests that use new style xdstp resource names do not specify
+			// an authority. These end up looking up an entry with the empty key
+			// in the authorities map. Having an entry with an empty key and
+			// empty configuration, results in these resources also using the
+			// top-level configuration, which is what we want mostly for our
+			// tests, unless explicitly specified by tests that use multiple
+			// authorities etc.
+			"": []byte(`{}`),
+		},
```

Comment:
Discussed offline. 

It's invalid to have resources with xdstp style names and without authority.  However we end up with these cases in some of the unit tests around per-resource type watches. 

It makes sense to not do the illegal thing in this test util API which seems to be most popular.

Easwar is planning to modify the tests to use a valid xdstp style resource name in this same PR if it's not too much work.

Reply:
> It's invalid to have resources with xdstp style names and without authority

Based on what we looked up, it is not technically *invalid* to have new style resource names with no authority. They simply will lookup an empty string in the `authorities` map of the bootstrap configuration. But we decided that the `e2e` package should not set up the bootstrap file by default with such an entry since this might hide some underlying issues. So, we decided that if some tests wants to have an entry in the authorities map with the empty, it can explicitly set it up that way.